src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapeGame.java

```diff
@@ -104,56 +108,32 @@ private boolean isPlayerOnGame(String playerName) {
 		
 		return false;
 	}
-	
-	private void distributePlayersPerTeam() {
-		int numberOfPlayers = _players.size();
-		int requiredPrisioners = (int) Math.round(
-				numberOfPlayers * ConfigManager.getInstance().getPrisionerRatio()
-		);
-		int requiredOfficers = (int) Math.round(
-				numberOfPlayers * ConfigManager.getInstance().getOfficerRatio()
-		);
-
-		Collections.shuffle(_players);
-		List<PrisonEscapePlayer> remainingPlayers = new ArrayList<>();
-
-		for (PrisonEscapePlayer player : _players) {
-			TeamPreference preference = player.getPreference();
-
-			if (preference == TeamPreference.POLICE && requiredOfficers != 0) {
-				_policeTeam.addMember(player);
-				requiredOfficers--;
-			} else if (preference == TeamPreference.PRISIONERS && requiredPrisioners != 0) {
-				_prisionersTeam.addMember(player);
-				requiredPrisioners--;
-			} else {
-				remainingPlayers.add(player);
-			}
-		}
 
-		for (PrisonEscapePlayer player : remainingPlayers) {
-
-			if (requiredPrisioners != 0) {
-				_prisionersTeam.addMember(player);
-				requiredPrisioners--;
-			} else {
-				_policeTeam.addMember(player);
-				requiredOfficers--;
-			}
+	/**
+	* @return      0 if success<br> 
+	* 				-1 if cannot start game
+	*/
+	public int startGame() {
+		if (_players.size() >= 3 && _phase instanceof Waiting) {
```

Comment:
You should be using the hasGameStarted method mentioned earlier.
The 3 value should be gotten using the ConfigManager. Create a new method named getMinimumPlayers or something like that.